vc_zoom/indico_vc_zoom/notifications.py

```diff
@@ -0,0 +1,44 @@
+# This file is part of the Indico plugins.
+# Copyright (C) 2020 CERN and ENEA
+#
+# The Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License;
+# see the LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+from indico.web.flask.templating import get_template_module
+from indico.core.notifications import make_email, send_email
+from indico.util.user import principal_from_identifier
+
+
+def notify_host_start_url(vc_room):
+    from indico_vc_zoom.plugin import ZoomPlugin
+
+    user = principal_from_identifier(vc_room.data['host'])
+    to_list = {user.email}
+
+    template_module = get_template_module(
+        'vc_zoom:emails/notify_start_url.html',
+        plugin=ZoomPlugin.instance,
+        vc_room=vc_room,
+        user=user
+    )
+
+    email = make_email(to_list, template=template_module, html=True)
+    send_email(email, None, 'Zoom')
+
+
+def notify_new_host(actor, vc_room):
+    from indico_vc_zoom.plugin import ZoomPlugin
+
+    template_module = get_template_module(
+        'vc_zoom:emails/notify_new_host.html',
+        plugin=ZoomPlugin.instance,
+        vc_room=vc_room,
+        actor=actor
+    )
+
+    new_host = principal_from_identifier(vc_room.data['host'])
+    email = make_email({new_host.email}, cc_list={actor.email}, template=template_module, html=True)
+    send_email(email, None, 'Zoom')
```

Comment:
`send_email(email, module='Zoom')` but `module` is not used unless there's an event (no place to log it otherwise), so:

```suggestion
    send_email(email)
```